pack.ps1

```diff
@@ -25,6 +27,15 @@ function createNuGetPackage([string]$pkgName, [string]$culture)
     $cultureFolder = [IO.Path]::Combine($localizationFolderName, $culture)
     Copy-Item -Path $cultureFolder -Destination $pkgCultureFolderPath -Recurse
     
+    echo "Copying '.props' template file .."
```

Comment:
echo "Copying '$pkgName.$pkgPropsExtension' ..."

Reply:
Okay

Reply:
done

Reply:
It's fine, because it's used once .. just leave it as it is